src/main/java/smile/identity/core/models/PartnerParams.java

```diff
@@ -0,0 +1,24 @@
+package smile.identity.core.models;
+
+import com.squareup.moshi.Json;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import smile.identity.core.enums.JobType;
+
+@Getter @Setter @NoArgsConstructor @AllArgsConstructor
+public class PartnerParams {
+    @Json(name = "job_type")
+    private JobType jobType;
+
+    @Json(name = "optional_info")
+    private String optionalInfo;
```

Comment:
`PartnerParams` can have more content.
ref: https://docs.smileidentity.com/further-reading/faqs/what-are-partner_params

the use of `optional_info` seems to be restrict that some. 

for clarity: will other fields be passed through to the request? 
that's what matters here.

Reply:
@tamssokari I can update `optionalInfo` to be a map so that way those using this sdk can pass in custom fields. 
I can then add a custom adapter that will know to take the fields of the `optionalInfo` map and add it to the body of `partnerParams` to look like this below 
```
{
	"user_id": "user",
	"job_id": "job",
	"job_type": 1,
	"custom_one": "custom_one",
	"custom_two": "custom_two"
}
```

Reply:
this sounds good to me.

and I'll like this pattern to be followed for all our server-side SDKs. so thanks for the trailblazing work you're doing here.